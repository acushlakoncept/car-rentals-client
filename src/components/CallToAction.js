import React from 'react';
import { BrowserRouter, Link } from 'react-router-dom';

export default function CallToAction() {
  return (
    <BrowserRouter>
      <section className="call-action">
        <h4>Be where you want to be on time</h4>
        <p>
          <Link className="btn btn-oval" to="/register">Book your ride</Link>
        </p>
      </section>
    </BrowserRouter>
  );
}
