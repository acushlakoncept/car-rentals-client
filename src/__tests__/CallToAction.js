import React from 'react';
import renderer from 'react-test-renderer';
import CallToAction from '../components/CallToAction';

it('renders CallToAction correctly', () => {
  const tree = renderer.create(<CallToAction />);
  expect(tree).toMatchSnapshot();
});
