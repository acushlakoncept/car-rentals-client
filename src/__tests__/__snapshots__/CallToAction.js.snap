// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CallToAction correctly 1`] = `
<section
  className="call-action"
>
  <h4>
    Be where you want to be on time
  </h4>
  <p>
    <a
      className="btn btn-oval"
      href="/register"
      onClick={[Function]}
    >
      Book your ride
    </a>
  </p>
</section>
`;
